service/dap/server.go

```diff
@@ -623,10 +626,55 @@ func (s *Server) onPauseRequest(request *dap.PauseRequest) { // TODO V0
 	s.sendNotYetImplementedErrorResponse(request.Request)
 }
 
-// onStackTraceRequest sends a not-yet-implemented error response.
+// onStackTraceRequest handles ‘stackTrace’ requests.
 // This is a mandatory request to support.
-func (s *Server) onStackTraceRequest(request *dap.StackTraceRequest) { // TODO V0
-	s.sendNotYetImplementedErrorResponse(request.Request)
+func (s *Server) onStackTraceRequest(request *dap.StackTraceRequest) {
+	// TODO(polina): support depth and cfg via launch/attach args
+
+	goroutineID := request.Arguments.ThreadId
+	// For feature parity with vscode-go, get depth (length of the returned list of Stackframes)
+	// from 'stackTraceDepth' in launch/attach arguments or default to 50.
+	// https://github.com/golang/vscode-go/blob/b4a484b59c4e3fe0966ea1056ddaeb5b77a83da0/src/debugAdapter/goDebug.ts#L373
+	// TODO(polina): Figure out why vscode-go does not specify DAP 'startFrame' + 'levels' value instead,
+	depth := 50
+	// Never set by vscode-go.
+	opts := api.StacktraceOptions(0)
+	// For feature parity with vscode-go, match its config defaults, which actually came from
+	// the many hardcoded places in this codebase. TODO(polina): make this a constant?
+	cfg := &api.LoadConfig{FollowPointers: true, MaxVariableRecurse: 1, MaxStringLen: 64, MaxArrayValues: 64, MaxStructFields: -1}
+
+	locs, err := s.debugger.Stacktrace(goroutineID, depth, opts, api.LoadConfigToProc(cfg))
+	if err != nil {
+		// No need to show to the user via pop-up as the error will be in CALL STACK panel
```

Comment:
I'm not sure what "pop-up" is referring to here. This is a server in Delve, we shouldn't be mentioning VSCode panels here